index.html

```diff
@@ -1412,6 +1412,91 @@ <h3>Biometrics</h3>
+        should be taken when securing an identifier system that supports these critical 
+        activities. As with all things, strong consideration for the appropriate and ethical 
+        use of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Environmental Considerations</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Principles as Applied to DID method development</h3>
+
+        <p>
+          The guiding principle that 
+          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+          web must be an environmentally sustainable platform</a> should be followed.
+          <br />
+          When implementing or utilizing a DID method, consideration should be given
+          to the environmental impacts of any underlying technologies. 
+        </p>
+
+        <p>
+          Utilizing or authoring DID methods that compromise on
+          the principles of security and / or control for other principles
+          such as environmental concerns should be very carefully 
+          considered in balance against alternative approaches that may
+          provide higher levels of security and protection.
+          The specific use case for a DID method will help determine the
+          optimal approach.  For matters that concern human rights, selection 
+          of approaches that are weighted towards privacy and control should 
+          outweigh other considerations.
+        </p>
+
+        <p class="issue">
+          Some members of the working group feel the the [[DID-RUBRIC]] should
+          include environmental considerations as defined in 
+          <a
+            href="https://github.com/w3c/did-rubric/pull/51">
+            this pull request</a> in order to properly evaluate aspects of a 
+            DID method that may apply to certain use cases.
```

Comment:
```suggestion
          Some members of the working group feel it important that the [[DID-RUBRIC]] 
          include environmental considerations as discussed in <a
          href="https://github.com/w3c/did-rubric/pull/51">this pull request</a> in
          order that users might appropriately evaluate and compare relevant aspects
          of DID methods that might be applicable to their use cases.
```

Reply:
This section doesn't feel appropriate here. Is the point to direct readers to the Rubric to find environmental criteria?

In fact, they should be directed to the rubric for any criteria in any category. The fact that "some members" feel this way also seems like a weird framing. 

Is there a lack of consensus on the use of the Rubric for these kinds of evaluations? I haven't heard anyone say the Rubric should exclude environmental considerations.